RecoTracker/MkFitCore/src/MkFinder.cc

```diff
@@ -584,85 +576,44 @@ namespace mkfit {
-                      st_z,
-                      q,
-                      L.hit_q(hi),
-                      ddq,
-                      dq,
-                      phi,
-                      L.hit_phi(hi),
-                      ddphi,
-                      dphi,
-                      tx,
-                      ty,
-                      tr,
-                      tphi,
-                      tz,
-                      tex,
-                      tey,
-                      ter,
-                      tephi,
-                      tez,
-                      hx,
-                      hy,
-                      hr,
-                      hphi,
-                      hz,
-                      hex,
-                      hey,
-                      her,
-                      hephi,
-                      hez,
-                      ht_dxy,
-                      ht_dz,
-                      ht_dphi,
-                      hchi2);
+                  // clang-format off
+                  printf("HITWINDOWSEL "
+                         "%d "
+                         "%d %d %d "
+                         "%d %d %d "
+                         "%6.3f %6.3f %6.3f %6.3f "
+                         "%d "
+                         "%d %d %d %d "
+                         "%d "
+                         "%d %d %d "
+                         "%6.3f %6.3f %6.3f "
+                         "%d %d %6.3f %6.3f "
+                         "%6.3f %6.3f %6.3f %6.3f %6.3f %6.3f %6.3f %6.3f "
+                         "%6.3f %6.3f %6.3f %6.3f %6.3f "
+                         "%6.6f %6.6f %6.6f %6.6f %6.6f "
+                         "%6.3f %6.3f %6.3f %6.3f %6.3f "
+                         "%6.6f %6.6f %6.6f %6.6f %6.6f "
+                         "%6.3f %6.3f %6.3f "
+                         "%6.3f"
+                         "\n",
+                         m_event->evtID(),
+                         L.layer_id(), L.is_barrel(), L.getOriginalHitIndex(hi),
+                         itrack, m_CandIdx(itrack, 0, 0), m_Label(itrack, 0, 0),
+                         1.0f / m_Par[iI].At(itrack, 3, 0), getEta(m_Par[iI].At(itrack, 5, 0)), m_Par[iI].At(itrack, 4, 0), m_Chi2(itrack, 0, 0),
+                         m_NFoundHits(itrack, 0, 0),
+                         m_SeedIdx(itrack, 0, 0), m_SeedLabel(itrack, 0, 0), m_SeedAlgo(itrack, 0, 0), thisseedmcid,
+                         mchid,
+                         st_isfindable, st_prodtype, st_label,
+                         st_pt, st_eta, st_phi,
+                         st_nhits, st_charge, st_r, st_z,
+                         q, L.hit_q(hi), ddq, dq, phi, L.hit_phi(hi), ddphi, dphi,
+                         tx, ty, tr, tphi, tz,
+                         tex, tey, ter, tephi, tez,
+                         hx, hy, hr, hphi, hz,
+                         hex, hey, her, hephi, hez,
+                         ht_dxy, ht_dz, ht_dphi,
+                         hchi2);
+                  // clang-format ofn
```

Comment:
typo?

Reply:
Yup, one backspace push too little, maybe I should oil it. Fixed.